gameros/airootfs/root/install.sh

```diff
@@ -2,46 +2,55 @@
 
 
 if [ $EUID -ne 0 ]; then
-	echo "$(basename $0) must be run as root"
-	exit 1
+    echo "$(basename $0) must be run as root"
+    exit 1
 fi
 
-#### Internet connection detection ####
-dhcpcd --background
-
+#### Wait for conenction or ask the user for configuration ####
 whiptail --infobox "Checking connection..." 10 50
 sleep 5
 
 while ! ( curl -Is https://gamer-os.github.io/ | head -1 | grep 200 > /dev/null ); do
-	whiptail --yesno "No internet connection detected. Please connect this computer\
-	 to the internet with a wired connection, wait a few seconds, then retry." 10 50\
-	 --yes-button "Retry"\
-	 --no-button "Exit"
-
-	if [ $? -ne 0 ]; then
-		 exit 1
-	fi
+    whiptail --yesno "No wired connection detected. Please connect this computer \
+     to the internet by configuring a new network." 10 50 \
+     --yes-button "Configure" \
+     --no-button "Exit"
+
+    if [ $? -ne 0 ]; then
+         exit 1
+    fi
+
+    nmtui
```

Comment:
Probably more user friendly to run `nmtui-connect` which jumps straight into selecting the connection.

Reply:
Ok. Didn't know of its existence. Will change it